docs/core/diagnostics/dotnet-stack.md

```diff
@@ -0,0 +1,116 @@
+---
+title: dotnet-stack tool - .NET Core
+description: Installing and using the dotnet-stack command-line tool.
+ms.date: 11/21/2019
+---
+# dotnet-stack performance analysis utility
+
+**This article applies to:** ✔️ .NET Core 3.0 SDK and later versions
+
+## Install dotnet-stack
+
+### Option 1 - Install as dotnet global tool:
+
+Install `dotnet-stack` [NuGet package](https://www.nuget.org/packages/dotnet-stack) with the [dotnet tool install](../tools/dotnet-tool-install.md) command:
+
+```dotnetcli
+dotnet tool install --global dotnet-stack
+```
+
+### Option 2 - Direct download:
+
+- Win ([x86](https://aka.ms/dotnet-stack/win-x86) | [x64](https://aka.ms/dotnet-stack/win-x64) | [arm](https://aka.ms/dotnet-stack/win-arm) | [arm-x64](https://aka.ms/dotnet-stack/win-arm64))
+- macOS ([x64](https://aka.ms/dotnet-stack/osx-x64))
+- Linux ([x64](https://aka.ms/dotnet-stack/linux-x64) | [arm](https://aka.ms/dotnet-stack/linux-arm) | [arm64](https://aka.ms/dotnet-stack/linux-arm64) | [musl-x64](https://aka.ms/dotnet-stack/linux-musl-x64) | [musl-arm64](https://aka.ms/dotnet-stack/linux-musl-arm64))
+
+## Synopsis
+
+```console
+dotnet-stack [-h, --help] [--version] <command>
+```
+
+## Description
+
+The `dotnet-stack` tool:
+
+* Is a cross-platform .NET Core tool.
+* Reports the managed stacks for all threads in .NET process.
+* Is built on [`EventPipe`](./eventpipe.md) of the .NET Core runtime.
+* Delivers the same experience on Windows, Linux, or macOS.
```

Comment:
I think the part about it being a cross-platform tool already makes this part pretty clear so we can leave this out.